### ISSUE TITLE
.zip contains 1810 still, not 1905.1 as advertised

### ISSUE THREAD

**Issue by devlocalca** [yuk7/CentWSL#9]
Greetings:

This page has a .zip file listed:
[https://github.com/yuk7/CentWSL/releases/tag/7.0.1905.1](https://github.com/yuk7/CentWSL/releases/tag/7.0.1905.1)

There is an indication that it contains **7.0.1905.1**, but when expanded and used, it's still 1810.

is this still a work in progress?

Would you be willing to share how '**rootfs.tar**' is created so I could roll my own?

---
I posted here asking that question:
[https://www.centos.org/forums/viewtopic.php?f=47&t=70999](https://www.centos.org/forums/viewtopic.php?f=47&t=70999)

**Comment by yuk7** [comment]
This is automatically built with CI.
Here is a script:[.travis.yml](https://github.com/yuk7/CentWSL/blob/master/.travis.yml)
and config is:[env.sh](https://github.com/yuk7/CentWSL/blob/master/env.sh)

**Comment by devlocalca** [comment]
This looks a little too complicated.

Is there a single command I can run to create a rootfs.tar of CentOS 7.6.1905?

The URL above still has 1810, not 1905.  Can this be corrected?

**Comment by yuk7** [comment]
I think the source has changed.
I'll check later.

**Comment by mishamosher** [comment]
CentOS does a "normal release" when upstream does (RedHat).

RHEL releases: [https://access.redhat.com/articles/3078](RHEL Release Dates)

CentOS releases: [https://wiki.centos.org/Download](CentOS Download)

Latest RHEL is 7.6, and CentOS updated to it and released on october 2018 (hence the name CentOS 1810).

Newer CentOS images like 1905 are simply a `yum update`d version of the "normal release".

TL;DR CentOS 1905 is the same as 1810, but with its packages updated to may 2019.

**Comment by mishamosher** [comment]
CentOS does a "normal release" when upstream does (RedHat).

RHEL releases: [RHEL Release Dates](https://access.redhat.com/articles/3078)

CentOS releases: [CentOS Download](https://wiki.centos.org/Download)

Latest RHEL is 7.6, and CentOS updated to it and released on october 2018 (hence the name CentOS 1810).

Newer CentOS images like 1905 are simply a `yum update`d version of the "normal release".

**TL;DR** CentOS 1905 is the same as 1810, but with its packages updated to may 2019.
